packages/nuxt/src/module.ts

```diff
@@ -21,6 +21,9 @@ export default defineNuxtModule<ModuleOptions>({
     const moduleOptions = {
       ...moduleOptionsParam,
       dynamicImportForServerEntry: moduleOptionsParam.dynamicImportForServerEntry !== false, // default: true
+      asyncFunctionReExports: moduleOptionsParam.asyncFunctionReExports
+        ? moduleOptionsParam.asyncFunctionReExports
+        : ['default', 'handler', 'server'],
```

Comment:
super-l: I find it slightly easier to read:

```suggestion
      asyncFunctionReExports: moduleOptionsParam.asyncFunctionReExports || ['default', 'handler', 'server'],
```